framework/raw-rsa-keyring.md

```diff
@@ -140,20 +140,20 @@ The keyring MUST attempt to decrypt the inputted encrypted data keys, in list or
 For each encrypted data key, the keyring MUST attempt to decrypt the encrypted data key into plaintext
 using RSA if and only if the following is true:
 
-- the encrypted data key's [key provider information](structures.md#key-provider-information)
+- The encrypted data key's [key provider information](structures.md#key-provider-information).
   has a value equal to this keyring's [key name](#key-name).
-- the encrypted data key's [key provider ID](structures.md#key-provider-id) has a value equal to
+- The encrypted data key's [key provider ID](structures.md#key-provider-id) has a value equal to
   this keyring's [key namespace](#key-namespace).
 
 The keyring performs RSA decryption with the following specifics:
 
-- this keyring's [private key](#private-key) is the RSA private key
-- this keyring's [padding scheme](#supported-padding-schemes) is the RSA padding scheme.
-- an encrypted data key's [ciphertext](structures.md#ciphertext) is the input ciphertext to RSA decryption.
+- This keyring's [private key](#private-key) is the RSA private key.
+- This keyring's [padding scheme](#supported-padding-schemes) is the RSA padding scheme.
+- An encrypted data key's [ciphertext](structures.md#ciphertext) is the input ciphertext to RSA decryption.
 
 If any decryption succeeds, this keyring MUST immediately return the input
 [decryption materials](structures.md#decryption-materials), modified in the following ways:
 
 - The output of RSA decryption is set as the decryption material's plaintext data key.
 
-If no decryption succeeds, this keyring MUST NOT make any update to the decryption materials.
+If no decryption succeeds, this keyring MUST NOT make any update to the decryption materials and MUST fail.
```

Comment:
Intentionally leaving vague so we can fix this under https://github.com/awslabs/aws-encryption-sdk-specification/issues/185

`MUST fail` wording provided by @mattsb42-aws